script.js

```diff
@@ -0,0 +1,35 @@
+//! All the needed html variables
+const LETTERS_CONTAINER = document.querySelector("#letters-container");
+const WORDS_CONTAINER = document.querySelector("#word-container");
+const HINT_CONTAINER = document.querySelector("#hint-container");
+
+const letters = "abcdefghijklmnopqrstuvwxyz".split("");
+
+// Go over each letter and creat it
+letters.forEach((letter) => {
+  createElement(
+    "button",
+    [{ attributeName: "letter-button", value: letter },
+    { attributeName: "class", value: "letter-button" },
+    { attributeName: "disable", value: "false" }
+  ],
+    letter,
+    LETTERS_CONTAINER
+  );
+});
+
+// Create elements and append them.
+function createElement(element, attributes, innerText, container) {
```

Comment:
Wouldn't it be better if you add event listeners here instead of what you do on line 31? Just a suggestion; otherwise, I can see this function being used throughout the project.